server/services/notificationCookieService.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response } from 'express'
+
+const COOKIE_NAME = 'ui-notification-banner'
+const key = (id: string) => `${COOKIE_NAME}-${id}`
+
+export const markAsDismissed = (res: Response, id: string) => {
+  const oneYear = 52 * 24 * 3600000
+  return res.cookie(key(id), 'dismissed', { maxAge: oneYear, httpOnly: true })
+}
+
+export const alreadyDismissed = (req: Request, id: string) => req.cookies && req.cookies[key(id)] === 'dismissed'
```

Comment:
don't think these need to be `export`ed here, since you've got the `export default` below?

Reply:
going with exporting them individually otherwise there are problems with the app.js require